common/actions/swap/actionTypes.ts

```diff
@@ -1,34 +1,55 @@
 import { TypeKeys } from './constants';
+
 export interface Pairs {
   ETHBTC: number;
   ETHREP: number;
   BTCETH: number;
   BTCREP: number;
 }
 
-export interface OriginKindSwapAction {
-  type: TypeKeys.SWAP_ORIGIN_KIND;
-  payload: string;
+export interface SwapInput {
```

Comment:
Action Creator Types should follow convention here: https://github.com/MyEtherWallet/MyEtherWallet#actiontypests